src/lib/utils.ts

```
@@ -1,6 +1,29 @@
+  return new Intl.NumberFormat(locale, {
+    style: 'currency',
+    currency: currency.replace('$', 'USD'),
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2
+  }).format(num);
+};
+
+export function formatRelativeTime(timestamp: string) {
+  if (!timestamp) return 'Unknown time';
+  const date = new Date(timestamp);
+  if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)